11-Syntax/03-Type-Level-Programming-Syntax/src/02-Basic-Syntax/01-Defining-Custom-Kinds.purs

```diff
@@ -19,11 +19,11 @@ foreign import data Value_Level_Instance2 :: Value_Level_Type_Kind
 -- despite the "foreign import" syntax!
 ----------------------
 
--- Using the value-level type Boolean as an example...
-data Boolean_ = True_Value | False_Value
+-- Using a Boolean-like value-level type as an example...
+data YesNo = Yes | No
 
-foreign import kind Boolean
+foreign import kind YesNo
```

Comment:
The way this is defined, it's hard to see the difference between then `YesNo` type and the `YesNo` kind. Elsewhere in this PR, you use `YesNoKind`, which is correct and which I'd prefer.

Reply:
I would say that this change should then propagate throughout and then we should have
```
foreign import kind YesNoK

foreign import data YesK :: YesNoK
foreign import data NoK :: YesNoK
```
as well, shouldn't we? Otherwise the `Yes/No` might get confused with the value-level instances

Reply:
I replied to your later comment, but stating it again here: yes, let's do it that way.